osmaxx-py/config/settings/common.py

```diff
@@ -381,8 +381,6 @@
     # to request access to osmaxx (access_denied page)
     'account_manager_username': env.str('ACCOUNT_MANAGER_USERNAME', default='admin'),
     'CONVERSION_SERVICE_URL': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_URL', default='http://localhost:8901/api/'),
-    'CONVERSION_SERVICE_USERNAME': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_USERNAME'),
-    'CONVERSION_SERVICE_PASSWORD': env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_PASSWORD'),
```

Comment:
would `env.str('DJANGO_OSMAXX_CONVERSION_SERVICE_PASSWORD', default='unknown_password)` be acceptable? (or something similar)